horovod/run/rendezvous/http_server.py

```diff
@@ -179,10 +179,13 @@ def __init__(self, verbose):
 
     # Rendezvous function finds a available port, create http socket,
     # and start listening loop to handle request
-    def start_server(self, host_alloc_plan):
+    def start_server(self, host_alloc_plan, pedl_provisioned_port=0):
```

Comment:
not the biggest fan of doubling up on defaults (default on the env variable as well as in this function signature). Can we remove one of them?.. (in the interest in speed, happy to have this happen at a later date)

Reply:
Good call, I got rid of most of the defaults that I put in now.